### PR TITLE
Support plugin

### PR DESCRIPTION
**Pull request by mkurek**
Sync supports from Ralph3.

### CODE REVIEW

**Review by szok — APPROVED** [review]

File: src/ralph_scrooge/tests/plugins/collect/test_ralph3_support.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+def _get_pricing_object_url(obj_id):
+    return 'http://ralph.allegro/api/base-objects/{}'.format(obj_id)
+
+
+class TestRalph3SupportPlugin(ScroogeTestCase):
+    def setUp(self):
+        self.today = date(2017, 2, 17)
+        self.daily_asset1 = DailyAssetInfoFactory()
+        self.daily_asset2 = DailyAssetInfoFactory()
+        self.daily_asset3 = DailyAssetInfoFactory.build()
```
Comment (xor-xor) [inline src/ralph_scrooge/tests/plugins/collect/test_ralph3_support.py]: Why this one has `.build()` while two others don't..? (just curious)

File: src/ralph_scrooge/tests/plugins/collect/test_ralph3_support.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        self.assertEquals(support.end, self.data['date_to'])
+        self.assertEquals(support.remarks, self.data['__str__'])
+        self.assertEquals(support.pricing_object.id, pricing_object.id)
+        cost = float(self.data['price']) / (
+            divide_to or len(self.data['base_objects'])
+        )
+        self.assertEquals(support.cost, cost)
+        self.assertEquals(support.forecast_cost, cost)
+
+    def test_update_support(self):
```
Comment (xor-xor) [inline src/ralph_scrooge/tests/plugins/collect/test_ralph3_support.py]: What about using some more descriptive names for test cases..? (i.e. what do you expect..? in what circumstances..? etc.)

File: src/ralph_scrooge/tests/plugins/collect/test_ralph3_support.py (hunk 1 above)
Comment (mkurek) [reply src/ralph_scrooge/tests/plugins/collect/test_ralph3_support.py]: I've moved it to the function where this is used (it's used only in one function). And the reason is: in this case, we want to create object but without saving it to DB to test object from ralph which is not present in Scrooge.

File: src/ralph_scrooge/plugins/collect/ralph3_support.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+
+def _get_object_id_from_url(url):
+    """
+    Extract object ID from Ralph API url.
+    """
+    return url.rstrip('/').rpartition('/')[2]
+
+
+def update_support(support):
```
Comment (xor-xor) [inline src/ralph_scrooge/plugins/collect/ralph3_support.py:24]: Consider adding some docstring here briefly describing what this fn does (e.g. what does it mean "update" in this context? what gets changed in `support`?).

**Review by xor-xor — APPROVED** [review]

**Pull request merged**